packages/manager/apps/hycu/src/constants.ts

```diff
@@ -9,4 +9,4 @@ export const packTypeLabel = {
 } as const;
 
 export const LICENSE_FILE_EXT = '.req';
-export const LICENSE_FILE_NAME_TEMPLATE = `license-hycu-{serviceName}.${LICENSE_FILE_EXT}`;
+export const LICENSE_FILE_NAME_TEMPLATE = `license-hycu-{serviceName}.dat`;
```

Comment:
Is it normal ? rebase issue ?

Reply:
Yes it's normal it's the download extension file and not he upload extension file